### ISSUE TITLE
credentials when running locally

### ISSUE THREAD

**Issue by mllewis** [nextml/NEXT#169]
When I try to run NEXT locally, I get a message in my browser saying:

Authentication is required.
[localhost] requires username and password
Your connection to this site is not private.

What credentials do I add here? I don't see anything in the documentation about this.

Thanks!

**Comment by stsievert** [comment]
This sounds like a network issue. What are the details of your setup? Operating system, web browser, docker version?

Try giving no credentials and see what happens.

**Comment by lalitkumarj** [comment]
Hi Molly

Just to confirm, are you going to http://local host:8000/home?

Also, is there any chance you are running a different server on your
computer for another application? This seems like the most likely culprit.

Finally, can you paste any output from the terminal window running NEXT?
Ie, the window with lots of colored lines floating by.

I'm confident we will be able to debug this.

Thanks!
Lalit

**Comment by stsievert** [comment]
From #178 it seems this issue is resolved. Is this true?

**Comment by mllewis** [comment]
No, I never got this to work (under time pressure, I ended up not running
locally). Sorry - I plan to try this again though for future iterations
though.

**Comment by stsievert** [comment]
No worries, just let us know what happens when you try again.
